corehq/apps/enterprise/forms.py

```diff
@@ -58,6 +134,50 @@ def __init__(self, *args, **kwargs):
                 ),
             )
         )
+
+        if DEFAULT_EXPORT_SETTINGS.enabled(self.domain):
+            self.helper.layout.append(
+                crispy.Fieldset(
+                    _("Edit Default Export Settings"),
+                    crispy.Div(
+                        crispy.HTML('<h4 style="font-weight: bold;">Form Data Exports</h4>')
```

Comment:
This h4 bold business is rather non-standard. I don't think we have a pattern for sub-fieldsets and am wary of creating one and encouraging nested forms. What would you think of making each of these groupings their own fieldset? "Edit Default Case Export Settings", "Edit Default OData Settings", etc.?

Reply:
Also translations are missing.

Reply:
I created separate fieldsets for each group like you suggested. I think that is the best short term solution.